1주차/1316.py

```diff
@@ -0,0 +1,33 @@
+#1316
+#똑같은 알파벳이 떨어져서 나타나면 그룹단어가 아님
+#똑같은 단어 거리가 절대값 1보다 크면 그룹단어가 아님 
+
+num=int(input())
+total=0
+
+
+for i in range(num):
+   a=input()
+   b=list(a)
+
+   for j in range(len(a)):
+    first=a.find(b[j])
+    last=a.rfind(b[j])
+    distance= last- first
+
+    group= True
+    if distance > 1:
+      for k in range(first+1,last):
+        if b[k] !=  b[j]:
+          group = False
+          break
+      if group == False:
+        break
+
+   if group==True:   
+    total+=1
```

Comment:
```suggestion
   else:   
    total+=1
```
우선, group 자체가 boolean값을 가지는 변수니까 굳이 `group == True`까지 써줄 필요 없이 `if group:` 으로만 적어줘도 돼요.
그리고 어차피 group이 False인 경우에는 항상 for문이 break로 끝나고, True인 경우 break를 만나지 않고 끝나니  for else문을 사용해봐도 좋을 것 같아요